test/queryselector.spec.ts

```diff
@@ -312,4 +359,100 @@ describe('querySelector', function () {
       expect(second).toEqual([]);
     });
   });
+
+  // this is the same tests for `$$eval` and `$$` as above, but with a handler
+  // that returns an array instead of a list of nodes.
+  describe('Custom query handler QueryAll can return Element[]', function () {
```

Comment:
The string passed to `describe()` is generally the name of the API that's being tested

Reply:
The public facing API that we are testing here is `registerCustomQueryHandler`, but that seems quite bit more general than what we are actually testing. I'll put `describe('QueryAll',...)` instead. WDYT?

Reply:
Yeah, names of internal APIs are fine if they are what's really being tested. Your suggestion SGTM!

Reply:
Great! Done.